lib/event_tracer/dynamo_db_log_worker.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+module EventTracer
+  class DynamoDBLogWorker
+    include ::Sidekiq::Worker
+
+    sidekiq_options retry: 1, queue: 'low'
+
+    TABLE_NAME = ENV.fetch('AWS_DYNAMODB_LOGGING_TABLE', 'logs')
```

Comment:
Could we define this table name and the app name in the app, then pass to the logger?

Reply:
That makes sense